### PR TITLE
Update rustdoc man page

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @rust-lang/docs 
r? @rust-lang/dev-tools

### CODE REVIEW

**Comment by QuietMisdreavus** [comment]
It still includes the section about output formats, including JSON output. `>_>`

File: man/rustdoc.1 [hunk 1]
```diff
@@ -60,23 +60,37 @@ pass arguments to the test runner
...
 \fB\-\-markdown\-no\-toc\fR
 don't include table of contents
 .TP
-\fB\-h\fR, \fB\-\-help\fR
-Print help
+\fB\-\-markdown\-no\-toc\fR
+don't include table of contents
+.TP
+\fB\-h\fR, \fB\-\-extend\-css\fR
+to redefine some css rules with a given file to generate doc with your own theme
```
Comment (killercup) [inline man/rustdoc.1]: Should be in sync with #41700

Comment (GuillaumeGomez) [inline man/rustdoc.1]: I didn't expect it to be merged before it so perfect. :D

**Comment by GuillaumeGomez** [comment]
Updated.

File: man/rustdoc.1 [hunk 2]
```diff
@@ -60,55 +59,50 @@ pass arguments to the test runner
...
 .TP
 \fB\-V\fR, \fB\-\-version\fR
 Print rustdoc's version
 
 .SH "OUTPUT FORMATS"
 
-The rustdoc tool can generate output in either an HTML or JSON format.
+The rustdoc tool can generate output in either an HTML or Markdown format.
```
Comment (QuietMisdreavus) [inline man/rustdoc.1]: Rustdoc can't create Markdown files, can't it? I was under the impression that the only thing rustdoc can output is the regular HTML pages.

Comment (GuillaumeGomez) [inline man/rustdoc.1]: Hum indeed, confused with the markdown input.

**Comment by GuillaumeGomez** [comment]
Updated.

File: man/rustdoc.1 [hunk 3]
```diff
@@ -60,55 +59,50 @@ pass arguments to the test runner
...
 URL to send code snippets to
 .TP
 \fB\-\-markdown\-no\-toc\fR
 don't include table of contents
 .TP
-\fB\-h\fR, \fB\-\-help\fR
-Print help
+\fB\-\-markdown\-no\-toc\fR
+don't include table of contents
+.TP
```
Comment (QuietMisdreavus) [inline man/rustdoc.1]: Duplicated lines

**Review by QuietMisdreavus — COMMENTED** [review]
Looks good! r=me after this.

**Pull request merged**